Stylesheets/Components/avatar.css

```diff
@@ -25,19 +23,19 @@
 
 .avatar-sm {
     width: 1.5rem;
-    height: 1.5rem;
+    height: auto;
```

Comment:
By default, height is set to `auto` so you can omit this.

Reply:
Thanks tanishq,
will remove it